Resources/doc/entity-listeners.rst

```diff
@@ -14,11 +32,43 @@ entity manager it should be registered with. Example:
             user_listener:
                 class: \UserListener
                 tags:
+                    # Minimal configuration
                     - { name: doctrine.orm.entity_listener }
+                    # Or, optionnally, you can give the entity manager name
+                    - { name: doctrine.orm.entity_listener, entity_manager: custom }
+    .. code-block:: xml
+
+        <?xml version="1.0" ?>
+
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+            <services>
+                <service id="user_listener" class="UserListener">
+                    <tag name="doctrine.orm.entity_listener" event="preUpdate" entity="App\Entity\User" />
+                    <tag
+                        name="doctrine.orm.entity_listener"
+                        entity_manager="custom"
+                    />
+                </service>
+            </services>
+        </container>
+        
+Starting with doctrine/orm 2.5 and Doctrine bundle 1.5.2, instead of registering the entity listener on the entity, you can declare all options from the service definition:
```

Comment:
```suggestion
Starting with doctrine/orm 2.5 and Doctrine bundle 1.5.2, instead of
registering the entity listener on the entity, you can declare all
options from the service definition:
```